docs/djdknativedigest.md

```diff
@@ -26,8 +26,10 @@
 
 This option enables or disables OpenSSL native cryptographic support for the Digest algorithm.
 
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restriction:** Pending the resolution of Github issue [#4530] (https://github.com/eclipse/openj9#4530), this option is disabled and has no effect. Currently, the Digest algorithm is not enabled by default.
```

Comment:
#4530 is already resolved, so this cannot be pending on the resolution of #4530. Also the link is incorrect, and there is a space between `] (`.